docs/docs/03-language-reference.md

```diff
@@ -1862,6 +1862,17 @@ let bucket = new awscdk.aws_s3.Bucket(
 );
 ```
 
+### 5.1.2 Type System
+
+Mapping JSII types to Wing types:
+
+| **JSII Type** |  **Wing Type** |
+|---------------|--------------------|
+| [class](https://aws.github.io/jsii/user-guides/language-support/assembly/#classes) | A [Wing class](#32-classes).<br> The [phase](#13-phase-modifiers) of the class will be `preflight` if the imported class is a [construct](https://github.com/aws/constructs) (derrived from `constructs.Construct`). Othersize the class will be phase independent.<br> By convention construct constructors have a `scope` and `id` as their constructor's first parameters. These will be used by Wing to define the default scope and id for new instances of this [preflight class](#33-preflight-classes) or explict scope and id using the `in` and `as` keywords. |
```

Comment:
```suggestion
| [class](https://aws.github.io/jsii/user-guides/language-support/assembly/#classes) | A [Wing class](#32-classes).<br> The [phase](#13-phase-modifiers) of the class will be `preflight` if the imported class is a [construct](https://github.com/aws/constructs) (derived from `constructs.Construct`). Otherwise the class will be phase independent.<br> By convention construct constructors have a `scope` and `id` as their constructor's first parameters. These will be used by Wing to define the default scope and id for new instances of this [preflight class](#33-preflight-classes) or explicit scope and id using the `in` and `as` keywords. |
```